src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -399,4 +402,41 @@ public void whenStreamTimeout0ThenInfiniteStreaming() throws Exception {
                 .anyMatch(streamBatch -> streamBatch.getMetadata() != null
                         && streamBatch.getMetadata().getDebug().equals("Stream timeout reached")));
     }
+
+    @Test(timeout = 15000)
+    public void whenResetCursorsThenStreamFromResetCursorOffset() throws Exception {
```

Comment:
@adyach 
For me this test doesn't work
I think the problem is that you expect 10 batches but in reality 11 batches come (10 + 1 service batch with "stream started" message)